### DISCUSSION TITLE
More advanced trend metric (graph) posibilities

### DISCUSSION THREAD

**Discussion by jsmit99** [laravel/nova-issues#5484] [Ideas & Feature Requests]
Add the ability to add multiple lines in a single trend metric (graph).

The current graph only allows for one line. It would be great being able to show multiple stats in the same chart.

![image](https://user-images.githubusercontent.com/82087876/233390578-91a2d99f-dec2-4ebc-9642-4419af25b7a6.png)

**Comment by chimit** [comment]
A multiline Trend chart would be very useful.

Also, a bar chart should be pretty easy to add because it's just another visualization of the existing Trend metric.

[Similar request](https://github.com/laravel/nova-issues/issues/1301).

 any plans on this?
